### PR TITLE
Add some additional AppKit controls

### PR DESCRIPTION
**Pull request by liscio**
This PR adds the following:

#### NSButton

- `states`: Sends state values for a "push on, push off" `NSButton` (like a checkbox) and other buttons that use state values.
- `state`: A `BindingTarget` to set the same as above

#### NSImageView

- `image`: Allows you to set the image through a `BindingTarget`

#### NSPopUpButton

- `selectedTags`: Sends the tag of the selected item as it changes
- `selectedTag`: Allows you to bind through a `BindingTarget` to set the above

#### NSTextField

- `textColor`: A `BindingTarget` allowing you to set the color of the text in a NSTextField
- `text`, `attributedText`: Wrappers for the {attributedS,s}tringValue `BindingTarget`s that aid in cross-platform coding

In addition, I added some tests for the above, and added a missing `extern` that was causing me some linker trouble in a project I was working on.

### CODE REVIEW

File: ReactiveCocoa/AppKit/NSTextField.swift [hunk 1]
```diff
@@ -22,4 +22,21 @@ extension Reactive where Base: NSTextField {
...
+	/// cross-platform compatibility
+	public var text: BindingTarget<String> {
+		return stringValue
+	}
+
+	/// Wraps the `stringValue` binding target from NSControl for
+	/// cross-platform compatibility
+	public var attributedText: BindingTarget<NSAttributedString> {
+		return attributedStringValue
+    }
```
Comment (mdiep) [inline ReactiveCocoa/AppKit/NSTextField.swift]: Indentation is off here.

File: ReactiveCocoa/AppKit/NSPopUpButton.swift [hunk 2]
```diff
@@ -43,4 +43,20 @@ extension Reactive where Base: NSPopUpButton {
...
 			.skipNil()
 	}
+
+
+	/// A signal of selected tags
+	public var selectedTags: Signal<Int, NoError> {
+		return self.integerValues
+			.map { [unowned base = self.base] _ -> Int in
+				return base.selectedTag()
+		}
```
Comment (mdiep) [inline ReactiveCocoa/AppKit/NSPopUpButton.swift]: Indentation of this closing `}` is off

**Comment by mdiep** [comment]
🤘 Its' great to see AppKit get some love.

Would you mind fixing the indentation while we wait for Travis to get fixed?

**Comment by liscio** [comment]
@mdiep No problem at all. I also have an NSSegmentedControl addition that I found kicking around in my repo that needs to get added as well.

**Comment by liscio** [comment]
@mdiep I suppose you didn't want additional tests for the `NSSegmentedControl` stuff I snuck in? ;)

No worries—I'll re-open with a few more late-breaking changes (like `NSButton`'s `alternateImage`) and hopefully a few basic tests. Tough to balance the "I should clean up & share these before I get too far off from the upstream" and, "I gotta finish this project I'm using it with…"

**Comment by mdiep** [comment]
> I suppose you didn't want additional tests for the NSSegmentedControl stuff I snuck in? ;)

😅 

Yeah, some tests would be great.

**Pull request merged**